src/Fable.Transforms/Replacements.fs

```diff
@@ -2474,6 +2474,17 @@ let hashSets (com: ICompiler) (ctx: Context) r t (i: CallInfo) (thisArg: Expr op
     // | "SymmetricExceptWith"
     | _ -> None
 
+let stacks (com: ICompiler) (ctx: Context) r t (i: CallInfo) (thisArg: Expr option) (args: Expr list) =
+    match i.CompiledName, thisArg, args with
+    | ".ctor", None, args ->
+        let moduleName, mangledName = getMangledNames i thisArg
+        Helper.LibCall(com, moduleName, mangledName, t, args, i.SignatureArgTypes, isJsConstructor=true, ?thisArg=thisArg, ?loc=r) |> Some
+    | ("Push" | "Pop" | "Peek" | "get_Count" | "Clear" | "Contains" |
+        "TryPeek" | "TryPop" | "ToArray" as meth), Some c, args ->
+        let moduleName, mangledName = getMangledNames i thisArg
+        Helper.LibCall(com, moduleName, mangledName, t, args, i.SignatureArgTypes, ?thisArg=thisArg, ?loc=r) |> Some
+    | _ -> None
```

Comment:
We have the `bclType` function to automatically map  types from the BCL (see `System.Text.StringBuilder` for example). Maybe could be used here too?

Reply:
Yep, `bclType` worked nicely!